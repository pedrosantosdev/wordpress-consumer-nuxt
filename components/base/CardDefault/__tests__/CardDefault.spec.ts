import { render } from '@testing-library/vue'
import CardDefaultVue from '@/components/base/CardDefault/CardDefault.vue'

describe('CardDefaultVue', () => {
	it('should render properly', () => {
		const wrapper = render(CardDefaultVue)
		expect(wrapper.html()).toMatchSnapshot()
	})
})
