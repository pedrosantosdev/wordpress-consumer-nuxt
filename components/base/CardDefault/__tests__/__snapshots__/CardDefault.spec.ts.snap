// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDefaultVue should render properly 1`] = `"<div class="lg:w-96 w-full sm:h-64 md:h-auto flex flex-col sm:flex-row p-2 sm:p-4 rounded-2xl shadow-lg gap-5 select-none dark:bg-gray-800 bg-white dark:bg-gray-800 dark:text-white"></div>"`;
